src/components/utils/components/Popup.tsx

```diff
@@ -1,5 +1,23 @@
 /* eslint-disable */
-export const Popup = ({ isOpen, onClose, children, onCloseAndSubmit }: any) => {
+import React, { useState } from 'react';
+import GEMINI_ICON from '../../../static/gemini-icon.png'; // Replace with the actual path to your Gemini icon
+
```

Comment:
Rather than changing the Popup.tsx fie, which is supposed to be as general as possible, create a AddGoogleCalanderPopup component that takes some extra props and uses Popup as a base